test/NuGet.Server.Tests/NuGet.Server.Tests.csproj

```diff
@@ -10,6 +11,8 @@
     <ProjectTypeGuids>{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}</ProjectTypeGuids>
     <TargetFrameworkVersion>v4.6</TargetFrameworkVersion>
     <TargetFrameworkProfile />
+    <NuGetPackageImportStamp>
+    </NuGetPackageImportStamp>
```

Comment:
> This shouldn't be happening anymore in 3.5.0, were you using a 2.x version of NuGet?

No, but maybe an old 3.x version. I'll remove it if this will no longer happen. Thanks!